### PR TITLE
Update Question2.py

### PR DESCRIPTION
**Pull request by satheesh15**

### CODE REVIEW

File: python/Question2.py [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
-    return answer
+    sum = 0
+    for i in range(0,len(trader)):
+	b = 0
+	for j in range(0,len(risk)):
+		if trader[i]>=risk[j]:
+			if bonus[j]>=b:
+				b = bonus[j]
+	sum = sum + b
+    return sum
```
Comment (madhur4127) [inline python/Question2.py]: you are posting your solution online for everyone else to see.

Comment (satheesh15) [reply python/Question2.py]: Does this code really works? :)

Comment (madhur4127) [reply python/Question2.py]: you can check yourself using the python test file provided.

Comment (satheesh15) [reply python/Question2.py]: I'm just confused..If u could, elaborate steps

**Pull request closed without merging**